extensions/2.0/Vendor/BENTLEY_materials_line_style/schema/material.BENTLEY_materials_line_style.schema.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "BENTLEY_materials_line_style extension",
+  "type": "object",
+  "description": "Specifies line width and dash pattern for CAD-style visualization.",
+  "allOf": [
+    { "$ref": "glTFProperty.schema.json" }
+  ],
+  "properties": {
+    "width": {
+      "type": "number",
+      "description": "Width of the line in screen pixels. Must be > 0.",
```

Comment:
Does it need  to be >0?  Can it have a default value of 1 if <=0?

Reply:
We should probably say the same thing in the point style PR I have up and recently edited.

Reply:
Also, maybe we should consider mentioning this fact in the README itself, not just the schema.

Reply:
^ (On this PR, not just the points one)